release-notes/9.0/linux-packages.json

```diff
@@ -0,0 +1,408 @@
+                        },
+                        {
+                            "id": "libssl",
+                            "name": "libssl3"
+                        },
+                        {
+                            "id": "libstdc++",
+                            "name": "libstdc++"
+                        },
+                        {
+                            "id": "icu",
+                            "name": "icu-libs"
+                        },
+                        {
+                            "id": "icu",
+                            "name": "icu-data-full"
+                        },
+                        {
+                            "id": "tzdata",
+                            "name": "tzdata"
+                        },
+                        {
+                            "id": "krb5",
+                            "name": "libgssapi-krb5-2"
+                        }
+                    ]
+                }
+            ]
+        },
+        {
+            "name": "Debian",
+            "install-commands":
+            [
+                {
+                    "run-under-sudo": true,
+                    "command-root": "apt-get",
+                    "command-parts": ["update"]
+                },
+                {
+                    "run-under-sudo": true,
+                    "command-root": "apt-get",
+                    "command-parts": ["install", "-y", "{packageName}"]
+                }
+            ],
+            "releases": [
+                {
+                    "name": "Debian 12 (Bookworm)",
+                    "release": "12",
+                    "packages": [
+                        {
+                            "id": "libc",
+                            "name": "libc6"
+                        },
+                        {
+                            "id": "libgcc",
+                            "name": "libgcc-s1"
+                        },
+                        {
+                            "id": "ca-certificates",
+                            "name": "ca-certificates"
+                        },
+                        {
+                            "id": "libssl",
+                            "name": "libssl3"
+                        },
+                        {
+                            "id": "libstdc++",
+                            "name": "libstdc++6"
+                        },
+                        {
+                            "id": "libicu",
+                            "name": "libicu72"
+                        },
+                        {
+                            "id": "tzdata",
+                            "name": "tzdata"
+                        },
+                        {
+                            "id": "krb5",
+                            "name": "libgssapi-krb5-2"
+                        }
+                    ]
+                },
+                {
+                    "name": "Debian 11 (Bullseye)",
```

Comment:
Should Bullseye be included here if it's EOL before .NET 9's release?

Reply:
Good call. Definitely ironic after our discussions about Debian 11 this week. Removed.